diff --git a/node_modules/@guidepup/guidepup/lib/macOS/VoiceOver/VoiceOver.js b/node_modules/@guidepup/guidepup/lib/macOS/VoiceOver/VoiceOver.js
index 25d6a70..3ef6493 100644
--- a/node_modules/@guidepup/guidepup/lib/macOS/VoiceOver/VoiceOver.js
+++ b/node_modules/@guidepup/guidepup/lib/macOS/VoiceOver/VoiceOver.js
@@ -128,6 +128,8 @@ class VoiceOver {
      * @returns {Promise<boolean>}
      */
     async detect() {
+        console.debug('isMacOS_1.isMacOS', isMacOS_1.isMacOS());
+        console.debug('supportsAppleScriptControl_1.supportsAppleScriptControl)()', await supportsAppleScriptControl_1.supportsAppleScriptControl());
         return (0, isMacOS_1.isMacOS)() && (await (0, supportsAppleScriptControl_1.supportsAppleScriptControl)());
     }
     /**
diff --git a/node_modules/@guidepup/guidepup/lib/macOS/VoiceOver/supportsAppleScriptControl/supportsAppleScriptControl.js b/node_modules/@guidepup/guidepup/lib/macOS/VoiceOver/supportsAppleScriptControl/supportsAppleScriptControl.js
index 10fc931..852cdc9 100644
--- a/node_modules/@guidepup/guidepup/lib/macOS/VoiceOver/supportsAppleScriptControl/supportsAppleScriptControl.js
+++ b/node_modules/@guidepup/guidepup/lib/macOS/VoiceOver/supportsAppleScriptControl/supportsAppleScriptControl.js
@@ -4,6 +4,8 @@ exports.supportsAppleScriptControl = supportsAppleScriptControl;
 const enabledDbFile_1 = require("./enabledDbFile");
 const enabledDefaults_1 = require("./enabledDefaults");
 async function supportsAppleScriptControl() {
+    console.debug('enabledDefaults_1.enabledDefaults()', await enabledDefaults_1.enabledDefaults());
+    console.debug('enabledDbFile_1.enabledDbFile)()', await enabledDbFile_1.enabledDbFile)());
     const results = await Promise.all([(0, enabledDefaults_1.enabledDefaults)(), (0, enabledDbFile_1.enabledDbFile)()]);
     return results.every((value) => value);
 }
