diff --git a/node_modules/@mdjs/mdjs-preview/src/MdJsPreview.js b/node_modules/@mdjs/mdjs-preview/src/MdJsPreview.js
index 59b0a83..1f2e66a 100644
--- a/node_modules/@mdjs/mdjs-preview/src/MdJsPreview.js
+++ b/node_modules/@mdjs/mdjs-preview/src/MdJsPreview.js
@@ -1,4 +1,4 @@
-import { LitElement, html, css } from 'lit-element';
+import { LitElement, html, css } from 'lit';
 import '@lion/accordion/define';

 import {
