diff --git a/node_modules/@open-wc/scoped-elements/types/index.d.ts b/node_modules/@open-wc/scoped-elements/types/index.d.ts
index 434e5cc..6bdfb06 100644
--- a/node_modules/@open-wc/scoped-elements/types/index.d.ts
+++ b/node_modules/@open-wc/scoped-elements/types/index.d.ts
@@ -5,4 +5,5 @@ export type ScopedElementsMap = {
         prototype: HTMLElement;
     };
 };
+export { ScopedElementsHost } from './src/types.js';
 //# sourceMappingURL=index.d.ts.map
