diff --git a/node_modules/@open-wc/semantic-dom-diff/chai-dom-diff-plugin.d.ts b/node_modules/@open-wc/semantic-dom-diff/chai-dom-diff-plugin.d.ts
index b3e4177..2b20a65 100644
--- a/node_modules/@open-wc/semantic-dom-diff/chai-dom-diff-plugin.d.ts
+++ b/node_modules/@open-wc/semantic-dom-diff/chai-dom-diff-plugin.d.ts
@@ -1,6 +1,6 @@
 /// <reference types="chai" />
 
-import { DiffOptions } from './get-diffable-html';
+import { DiffOptions } from './get-diffable-html.js';
 
 declare global {
   namespace Chai {
