diff --git a/node_modules/@rocket/cli/src/RocketStart.js b/node_modules/@rocket/cli/src/RocketStart.js
index c7bd339..4a751a6 100755
--- a/node_modules/@rocket/cli/src/RocketStart.js
+++ b/node_modules/@rocket/cli/src/RocketStart.js
@@ -96,11 +96,13 @@ export class RocketStart {
       await this.eleventy.write();
     }
 
+
     /** @type {DevServerConfig} */
     const devServerConfig = metaConfigToWebDevServerConfig(
       {
         nodeResolve: true,
-        watch: this.config.watch !== undefined ? this.config.watch : true,
+        // This prevents endless watch loops (manual refreshes are needed now, but this is a good tradeoff)
+        watch: false, // this.config.watch !== undefined ? this.config.watch : true,
         rootDir: this.config.outputDevDir,
         open: true,
         clearTerminalOnReload: false,
