diff --git a/node_modules/singleton-manager/src/SingletonManagerClass.js b/node_modules/singleton-manager/src/SingletonManagerClass.js
index 6a10af5..f47bff9 100644
--- a/node_modules/singleton-manager/src/SingletonManagerClass.js
+++ b/node_modules/singleton-manager/src/SingletonManagerClass.js
@@ -3,7 +3,7 @@ const sym = Symbol.for('lion::SingletonManagerClassStorage');
 export class SingletonManagerClass {
   constructor() {
     /** @protected */
-    this._map = window[sym] ? window[sym] : (window[sym] = new Map());
+    this._map = globalThis[sym] ? globalThis[sym] : (globalThis[sym] = new Map());
   }
 
   /**
